zipline/pipeline/factors/technical.py

```diff
@@ -162,6 +162,36 @@ def compute(self, today, assets, out, close, volume):
         out[:] = nanmean(close * volume, axis=0)
 
 
+class RateOfChangePercentage(CustomFactor):
+    """
+    Rate of change Percentage
+    ROC measures the percentage change in price from one period to the next.
+    The ROC calculation compares the current price with the price `n`
+    periods ago.
+    Formula for calculation: ((price - prevPrice) / prevPrice) * 100
+    price - the current price
+    prevPrice - the price n days ago, equals window length
+
+    **Default Inputs**: [USEquityPricing.close]
```

Comment:
This factor seems like it is sufficiently general, how do you feel about removing the defaults here?